docs/wallets/ledger.md

```diff
@@ -0,0 +1,103 @@
+# Ledger Hardware Wallets
+
+You can use your [Ledger hardware wallets](https://www.ledger.com/) to securely manage your Testnet CRO assets. Ledger devices are hardware wallets which stores your private keys safely in the device and can be used in even unsafe PCs.
+
+## Pre-requisites
+
+- An initialized Ledger Nano device
+- Install the latest firmware on your device
+- Install Ledger Live on your PC
+- Download our latest [release binary](https://chain.crypto.com/docs/getting-started/croeseid-testnet.html)
+
+## Install the Ledger app
+
+There are currently two ways to install the Ledger app for Crypto.com Chain:
+
+1. [Ledger Live (Preferred)](#1-install-via-ledger-live)
+1. [Build from source code](#2-build-from-source-code)
+
+### 1. Install via Ledger Live
+
+:::tip
+Crypto.com Chain Ledger application is being reviewed at the moment. Please use the Cosmos application for the time being, which has the same functionalities except for the supported address derivation path.
+:::
+
+1. Open "Manager" in Ledger Live.
+1. Connect and unlock your Ledger devices.
+1. If asked, allow Ledger manager on your Ledger device.
+1. Search for Cosmos (ATOM) in the app catalog in Ledger Live.
+1. Click the Install button to install the app.
+1. Your device will display "Processing". Wait until the installing completed.
+1. Your device will have a new app named "Cosmos"
+
+### 2. Build from source code
+
+:::danger
+We HIGHLY DISCOURAGED using this method to install the Ledger app because it requires putting your Ledger device into development mode and loaded with an unsafe mnemonics. NEVER use this on a Ledger with funds and NEVER use the development Ledger devices on any production blockchain network.
+:::
+
+You can follow the [instructions](https://github.com/crypto-com/ledger-crypto#using-a-real-device) to build from source code and install the application to your development purpose ONLY Ledger device.
+
+## Generate Address
+
+1. Connect and unlock your Ledger device
+1. Open "Cosmos" or "Crypto.com Chain" app on the device
+1. Run in your terminal
+    ```bash
+    $ ./chain-maind keys add [name] --coin-type=[coin-type] --account=[account-index] --ledger --keyring-backend=test
+    ```
+
+    - `name` is an arbitrary name for your new key
+    - `coin-type` is the Coin type. Depends on the application you installed, please use different types:
+        | App | Coin type |
+        | --- | --- |
+        | Cosmos| `118` |
+        | Crypto.com Chain | `394` |
+    - `account-index` can be any integer inclusively starting from `0` to `2147483648`
```

Comment:
Can add sth like: 

`keyring-backend` is the keyring's storage backend, we will use `--keyring-backend test` here for simplicity.